src/Dashboard/App_Start/AppModule.cs

```diff
@@ -35,46 +37,81 @@ public override void Load()
             }
 
             CloudBlobClient blobClient = account.CreateCloudBlobClient();
-            CloudQueueClient queueClient = account.CreateCloudQueueClient();
-
             Bind<CloudStorageAccount>().ToConstant(account);
             Bind<CloudBlobClient>().ToConstant(blobClient);
-            Bind<CloudQueueClient>().ToConstant(queueClient);
-
-            Bind<IHostVersionReader>().To<HostVersionReader>();
-            Bind<IDashboardVersionManager>().To<DashboardVersionManager>();
-            Bind<IFunctionInstanceLookup>().To<FunctionInstanceLookup>();
-            Bind<IFunctionInstanceLogger>().To<FunctionInstanceLogger>();
-            Bind<IHostIndexManager>().To<HostIndexManager>();
-            Bind<IFunctionIndexVersionManager>().To<FunctionIndexVersionManager>();
-            Bind<IFunctionIndexManager>().To<FunctionIndexManager>();
-            Bind<IFunctionLookup>().To<FunctionLookup>();
-            Bind<IFunctionIndexReader>().To<FunctionIndexReader>();
-            Bind<IHeartbeatValidityMonitor>().To<HeartbeatValidityMonitor>();
-            Bind<IHeartbeatMonitor>().To<HeartbeatMonitor>();
-            Bind<Cache>().ToConstant(HttpRuntime.Cache);
-            Bind<IFunctionStatisticsReader>().To<FunctionStatisticsReader>();
-            Bind<IFunctionStatisticsWriter>().To<FunctionStatisticsWriter>();
-            Bind<IRecentInvocationIndexReader>().To<RecentInvocationIndexReader>();
-            Bind<IRecentInvocationIndexWriter>().To<RecentInvocationIndexWriter>();
-            Bind<IRecentInvocationIndexByFunctionReader>().To<RecentInvocationIndexByFunctionReader>();
-            Bind<IRecentInvocationIndexByFunctionWriter>().To<RecentInvocationIndexByFunctionWriter>();
-            Bind<IRecentInvocationIndexByJobRunReader>().To<RecentInvocationIndexByJobRunReader>();
-            Bind<IRecentInvocationIndexByJobRunWriter>().To<RecentInvocationIndexByJobRunWriter>();
-            Bind<IRecentInvocationIndexByParentReader>().To<RecentInvocationIndexByParentReader>();
-            Bind<IRecentInvocationIndexByParentWriter>().To<RecentInvocationIndexByParentWriter>();
-            Bind<IHostMessageSender>().To<HostMessageSender>();
-            Bind<IPersistentQueueReader<PersistentQueueMessage>>().To<PersistentQueueReader<PersistentQueueMessage>>();
-            Bind<IFunctionQueuedLogger>().To<FunctionInstanceLogger>();
-            Bind<IHostIndexer>().To<HostIndexer>();
-            Bind<IFunctionIndexer>().To<FunctionIndexer>();
-            Bind<IIndexer>().To<UpgradeIndexer>();
-            Bind<IInvoker>().To<Invoker>();
-            Bind<IAbortRequestLogger>().To<AbortRequestLogger>();
-            Bind<IAborter>().To<Aborter>();
-
-            Bind<IIndexerLogWriter>().To<IndexerBlobLogWriter>();
-            Bind<IIndexerLogReader>().To<IndexerBlobLogReader>();
+
+            string tableLog = ConfigurationManager.AppSettings["AzureWebJobsLogTableName"];
```

Comment:
Since we're going to need to move to a more flexible multi-table approach, I don't think we should have a setting that indicates a table name. I think the table structure should be completely internal to the FastTableReader. If in the short term it only uses a single table, that should be an implementation detail, and we need to make sure all the new APIs will support the future.

If you don't have time now to implement the multi-table approach, we should have a bug to address that, so you can do it later.